android/quest/src/main/java/org/smartregister/fhircore/quest/data/DataMigration.kt

```diff
@@ -220,6 +222,7 @@ constructor(
       }
     }
     preferenceDataStore.write(PreferenceDataStore.MIGRATION_VERSION, maxVersion)
+    defaultRepository.sharedPreferencesHelper.write(SharedPreferenceKey.DATA_MIGRATION.name, maxVersion)
```

Comment:
Delete this. PreferenceDataStore is used above.

Reply:
fixed